test/docker/fs/etc/colonus/anax-combined.config.tmpl

```diff
@@ -10,6 +10,7 @@
         "PublicKeyPath": "/root/.colonus/mtn-publicKey.pem",
         "DefaultServiceRegistrationRAM": 1024,
         "ExchangeURL": "${EXCH_APP_HOST}/",
+        "CACertsPath": "/certs/css.crt",
```

Comment:
I thought I made this comment before. CACertsPath is already specified in this file, why not delete the other one? This problem exists on the other similar files too.